app/components/request_and_pickup_button_component.rb

```diff
@@ -23,7 +23,7 @@ def default_pickup_library
   end
 
   def single_checked_out_item?
-    @current_request.holdings_object.single_checked_out_item?
+    @current_request.holdings_object.all.one? && @current_request.holdings_object.all.first.checked_out?
```

Comment:
Should we make Holdings an Enumerable so we can use `one?` or `all?` directly on that object? `all.first` and `all.one?` read a little funky.

Reply:
yeah, I had the same thought. will do

Reply:
sigh. doing this means you have to mock `each` in every test, because these things are all just regular `double`s...

Reply:
i guess i could change all the `all:` to `each:` ...

Reply:
@cbeer is there a smart way to do this I'm missing?

Reply:
added some TODOs about it.